### DISCUSSION TITLE
we need this for 1.35 and/or 1.37 please

### DISCUSSION THREAD

**Discussion by Oscillosirus** [lixiangwuxian/Quest-Discord-Presence#2] [General]

**Comment by lixiangwuxian** [comment]
Still learning how to use scotland2 and new beatsaber hook library. Modding in 1.35 changes a lot😭

**Comment by lixiangwuxian** [comment]
Done

**Reply by Oscillosirus** [reply]
thx, you're a bloody legend

**Comment by dotheboogey678** [comment]
i know this is old but are you able to update it for 1.40.8?

**Reply by lixiangwuxian** [reply]
> i know this is old but are you able to update it for 1.40.8?

Here, @CatsaCode did this
https://github.com/CatsaCode/Quest-Discord-Presence
